src/rdkafka_conf.c

```diff
@@ -4304,25 +4363,107 @@ int unittest_conf (void) {
                                 "!1.2.3.4.5!!! a", NULL, 0);
         RD_UT_ASSERT(res == RD_KAFKA_CONF_OK, "%d", res);
 
+        /* Verify that sasl.oauthbearer.* string-safing works */
+        res = rd_kafka_conf_set(conf, "sasl.oauthbearer.method",
+                                "oidc", NULL, 0);
+        RD_UT_ASSERT(res == RD_KAFKA_CONF_OK, "%d", res);
```

Comment:
The conf unittests are for testing the conf framework, and not individual configuration properties.
The config properties are to be tested by their corresponding tests, so in this case you would create a tests/01??_oauthbearer_oidc.c (or cpp) file that sets up a client for OIDC.

There's also generally no need to conf_get() and verify each individual config property, since that functionality is verified by the unittests and the 0004 tests.